### ISSUE TITLE
make fails when building zbasic

### ISSUE THREAD

**Issue by pfrankis** [ZipCPU/zbasic#4]
I am following the instructions on : http://zipcpu.com/zipcpu/2018/02/12/zbasic-intro.html

when I try to run `make` under the `zbasic` directory, I get the following:

```Building dependency file(s) 
zip-gcc -O3 -c hello.c -o obj-zip/hello.o
make[1]: *** No rule to make target '/home/dan/work/rnd/zipcpu/sw/install/cross-tools/zip/lib/libg.a', needed by 'hello'.  Stop.
Makefile:182: recipe for target 'sw-board' failed
make: *** [sw-board] Error 2```

**Comment by ZipCPU** [comment]
This looks like a tools issue.  Did you manage to install the ZipCPU toolchain without any issues?  Including newlib?  If so, when you installed newlib, can you tell me what prefix was used?

**Comment by pfrankis** [comment]
Yes, I followed the steps here: http://zipcpu.com/zipcpu/2018/01/31/cpu-build.html
and didn't have any issues ( I went as far successfully running simtest before switching back to the zbasic instructions).

I just tried running this command again:
`sudo apt install flex bison libbison-dev libgmp-dev libmpfr-dev libmpc-dev bc`

but it said that everything was already up to date.

**Comment by ZipCPU** [comment]
Can you run `zip-gcc -v` and post the output here?

Thanks!

**Comment by pfrankis** [comment]
```Using built-in specs.
COLLECT_GCC=zip-gcc
COLLECT_LTO_WRAPPER=/mnt/c/work/git/references/zipCPU/zipcpu/sw/install/cross-tools/libexec/gcc/zip/6.2.0/lto-wrapper
Target: zip
Configured with: ../gcc-6.2.0-zip/configure --with-gas --prefix=/mnt/c/work/git/references/zipCPU/zipcpu/sw/install/cross-tools --target=zip --with-pkgversion=zip-gcc-191224 --disable-multilib --disable-threads --disable-tls --enable-checking --disable-nls --with-newlib
Thread model: single
gcc version 6.2.0 (zip-gcc-191224)```

**Comment by ZipCPU** [comment]
Hmm ... that doesn't look like it should have a problem.

I just ran `make hello` from within the sw/board directory, and I saw a different command completely.  Can you tell me if this is the command you are seeing?

```text
% zip-gcc -O3 -I../zlib -I../../rtl -T board.ld -L../zlib obj-zip/hello.o -lc -lzbasic -lgcc -o hello
```

This is from a fresh build from github.

Dan

**Comment by pfrankis** [comment]
yeah, I don't know what's going on.  I started from a fresh build from github and I still get the same command I have above, not the one you're seeing:
`zip-gcc -O3 -c hello.c -o obj-zip/hello.o`

**Comment by ZipCPU** [comment]
This is now fixed in [92c59a2](https://github.com/ZipCPU/zbasic/commit/92c59a2a094820a749edee1df4f3002cc712abe2).

Thanks or your help and reporting!
